os-server/src/main/resources/db/migration/h2/V1_0_2__update.sql

```diff
@@ -0,0 +1,13 @@
+-- -----------------------------------------------------
```

Comment:
why this is added in this PR ?

Reply:
actually, it isn't relevant to this task, but as Hakob and I are working together on DB I've added this one too. It's for his needs.